test/extensions/filters/http/alternate_protocols_cache/filter_integration_test.cc

```diff
@@ -82,19 +109,27 @@ TEST_P(FilterIntegrationTest, AltSvc) {
   Http::TestRequestHeaderMapImpl request_headers{
       {":method", "POST"},    {":path", "/test/long/url"}, {":scheme", "http"},
       {":authority", "host"}, {"x-lyft-user-id", "123"},   {"x-forwarded-for", "10.0.0.1"}};
-  int port = fake_upstreams_[0]->localAddress()->ip()->port();
+  int port = fake_upstreams_[1]->localAddress()->ip()->port();
```

Comment:
fake_upstreams_[0] also uses this port, so no need to change?

Reply:
yeah but I'm hoping some day they won't share a port, and I _think_ we want the alt-svc header using the quic port.